docs/tutorials/google/calibration_faq.md

```diff
@@ -0,0 +1,311 @@
+each gate. This will improve the accuracy of the overall moment without adding additional duration to the circuit (the newly
+added Z gates will be merged with the already existing Z gates after sending circuits to the QCS.)
+
+The [two supported techniques](./calibration_api.ipynb) are [Floquet calibration](./floquet_calibration_example.ipynb)
+and [XEB calibration](./xeb_calibration_example.ipynb), both of which are advanced alpha features.
+
+## What is the Floquet calibration protocol?
+
+Floquet calibration protocol allows for determining the parameters of entangling gates rapidly and precisely, which unlocks
+the ability to compensate for systematic gate errors caused by drifts and fluctuations. Like XEB, it is robust to state 
+preparation and measurement errors. Floquet calibration is based on the idea that an entangling gate can be determined by
+the eigenvalues of the composite gates consisting of the entangling gate and different single-qubit gates, where each choice
+of single-qubit gates provides a snapshot of the entangling gate. There are two stages in Floquet calibration, characterization
+and compensation. Currently, only the single-qubit Z phases before and after the two-qubit gate are compensated. The 
+compensation pulses are combined with the gate pulses; therefore, they do not increase the circuit implementation time. In 
+structured circuits, two-qubit gates in the same moment should be calibrated simultaneously to combat crosstalks if possible.
+Floquet calibration is really fast, which allows one to calibrate multiple moment structures within budgeted time.
+
+## Which gates are supported by Floquet and XEB calibrations?
+
+Floquet calibrations specified by `cirq_google.FloquetPhasedFSimCalibrationOptions` supports characterization of four out of
+five unitary parameters θ, ζ, γ and φ of the `cirq.FSim(theta=pi / 4, phi=0)` gate (inverse of the `cirq.SQRT_ISWAP` gate) 
+presently.  The Cirq compiler can use Z gates to compensate for the ζ, χ, γ parameters.  The compiler can also compile `cirq.SQRT_ISWAP`
+gate into its inverse so that it can be used in calibrated circuits and take advantage of the characterization.
+
+The XEB calibration specified by `cirq_google.LocalXEBPhasedFSimCalibrationOptions` supports `cirq.SQRT_ISWAP` and its inverse.  
+It characterizes all five unitary parameters and also provides the fidelity estimates for each of the calibrated gates. However, 
+the compiler can only adjust for three of the five parameters by inserting Z gates.
+
+This version of XEB is using the client-side Cirq characterization backed by 
+`cirq.experiments.xeb_fitting.characterize_phased_fsim_parameters_with_xeb_by_pair` and when used directly it can support 
+arbitrary gates, including composite ones. However, the compiler which adjusts for ζ, χ, γ angles available in cirq_google 
+package supports only `cirq.SQRT_ISWAP` with its inverse.
```

Comment:
This sentence is no longer true, the support for sycamore gate was added a while back. However, it was tested only through unit tests until now.

Reply:
Thanks for noting this! Updated to reflect the support for `cg.SYC` as well.